laravel/app/Http/Controllers/ScheduleDataController.php

```diff
@@ -527,50 +396,33 @@ public function updateMilestones($id)
             ->orderBy("last_name")
             ->get();
 
-        $datas = compact("data1", "data2", "data3");
+        $resident_choices = $resident_data;
 
-        return view("schedules.resident.milestone", compact("id", "milestones", "datas", "anesthesiologists"));
+        return view(
+            "schedules.resident.milestone",
+            compact("id", "milestones", "resident_choices", "anesthesiologists")
+        );
     }
 
     public function notifyResidentOverwrittenPreferences($toName, $toEmail, $residentName, $date, $overwrittenChoices)
     {
-        $choice = "";
-        if ($overwrittenChoices[0] == 1) {
-            $choice = "1";
-        }
-
-        if ($overwrittenChoices[1] == 2) {
-            $choice = $choice . " 2";
-        }
-
-        if ($overwrittenChoices[2] == 3) {
-            $choice = $choice . " 3";
-        }
+        $choice = implode(", ", $overwrittenChoices);
+        $choice .= " ";
```

Comment:
I don't think we need to add the space here, since we have one prior to "Overwritten" on the following line.

Reply:
You can add the space in line 414: `$subject = " REMODEL...";`

Reply:
I believe this line should be removed. Otherwise, the choice will have two spaces after it.